src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/CaseloadItem.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+import java.time.LocalDate
+
+@Schema(description = "Additional information pertinent to CVL")
+data class CvlFields(
```

Comment:
key to store CVL specific fields- this can hopefully be restructured in the future